### ISSUE TITLE
Cannot copy output from console into editors

### ISSUE THREAD

**Issue by herambgadgil** [rstudio/rstudio#7921]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.4.887
    OS Version      : Windows
    R Version       : 3.6.3

### Steps to reproduce the problem
Execute this on console `print("Testing the copy-paste functionality")`
Copy it either with keyboard shortcut `Ctrl + C` or select, right-click and click on `Copy`
Try to paste it in editor with `Ctrl + V` or right-click menu. The paste option is de-activated. 

### Describe the problem in detail
Not able to copy and paste the outputs from console into editors or anywhere. 

### Describe the behavior you expected
The copied content from console should be pasted.

- [X] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [X] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [X] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by herambgadgil** [comment]
Quick addition to this - Its not just for `Console` window. Same bug encountered on `Terminal` window as well

**Comment by jmcphers** [comment]
Thanks for trying the dailies! However, I couldn't reproduce this (Windows 10, RStudio 1.4.894). How are you selecting the text between the first two steps?

**Comment by herambgadgil** [comment]
@jmcphers - I tried restarting my system and even I am not facing the issue now in `1.4.887`. I think it was my system's error. Its amusing though, because I had tried to Copy + Paste from other software and editors and it was working. I restarted RStudio thrice to be sure that I am able to re-iterate it. The only thing that I did not look for is system restart! :-)

Sorry about that. Next time I will add this step in the verification stage to be sure. I think we can close this bug. It was a false positive.

Thanks!
Heramb
